components/__tests__/search-form.test.tsx

```diff
@@ -2,7 +2,8 @@ import React from "react";
 
 import { SearchForm } from "../search-form";
 
-import { render } from "@testing-library/react";
+import { fireEvent, render, screen } from "@testing-library/react";
```

Comment:
Please update all the `fireEvent` usages in this file so that you don't need to import it here, Mbak @ekamuktia. 😁🙏